docker/config.hcl

```diff
@@ -0,0 +1,65 @@
+disable_mlock = true
+
+controller {
+  name = "demo-controller-1"
+  description = "A controller for a demo!"
+
+  database {
+    url = "postgresql://postgres:postgres@0.0.0.0:5432/postgres?sslmode=disable"
```

Comment:
Going to update this with the `env://` pattern to allow easy overrides from `-e` flag in docker run.

Reply:
Updated